lib/api/objectPutACL.js

```diff
@@ -119,6 +120,16 @@ export default function objectPutACL(authInfo, request, log, cb) {
             let usersIdentifiedByEmail = [];
             let usersIdentifiedByGroup = [];
             let usersIdentifiedByID = [];
+
+            // If grants set by xml and owner ID is incorrectly overwritten
```

Comment:
what do you mean by overwritten?  do you just mean they got the owner id wrong?  If so, please state that.